doc/codingGuidelines.md

```diff
@@ -148,16 +148,22 @@ In general, macros should be avoided (see [Modern C++](https://docs.google.com/d
 * Comments for users of classes and functions must be written in headers files. Comments in definition files are meant for contributors and maintainers.
 
 ### Include directive
+For source files (.cpp) with an associated header file (.h) that resides in the same directory, it should be `#include`'d with double quotes and no path.  This formatting should be followed regardless of with whether the associated header is public or private. For example:
```

Comment:
Ah, sorry, this now reads "regardless of with whether" as opposed to "regardless of whether". :)